RELEASE_CHECKLIST.md

```diff
@@ -26,6 +26,9 @@ This is a list of the things that need to happen during a release.
 1. Run `cargo update`.
 1. Run `cargo test`.
 1. Run `cargo build`.
+1. Copy `README.md` to `npm/README.md`
```

Comment:
It'd be nice if all of this could be automated with a script, but we can do that later, so this is fine for now.